front/app/components/PostShowComponents/Comments/ChildCommentForm.tsx

```diff
@@ -167,10 +172,16 @@ class ChildCommentForm extends PureComponent<Props & InjectedIntlProps, State> {
   }
 
   onChange = (inputValue: string) => {
+    const { tag } = this.state;
+    const tagLength = tag ? tag.length : 0;
+    const inputValueWithoutTag = inputValue.slice(tagLength);
```

Comment:
When replying to someone, the comment gets filled automatically with @parent author, so checking here for emptiness with that part removed.